src/main/java/org/reaktivity/nukleus/sse/internal/types/codec/SseEventFW.java

```diff
@@ -276,5 +280,36 @@ public Builder type(
             return this;
         }
 
+        public Builder comment(boolean enabled)
```

Comment:
This should receive a `DirectBuffer` same as other similar methods, even if our initial usecase is just the empty buffer.